dashboard/src/components/Header/Menu.tsx

```diff
@@ -27,8 +29,26 @@ function Menu({ clusters, appVersion, logout }: IContextSelectorProps) {
   // Control when users click outside
   const ref = useRef(null);
   useOutsideClick(setOpen, [ref], open);
+  const [isDarkModeEnabled, setIsDarkModeEnabled] = useState(
+    localStorage.getItem("theme") === SupportedThemes.dark ? true : false,
+  );
 
   const toggleOpen = () => setOpen(!open);
+  const toggleDarkMode = () => {
+    localStorage.setItem(
+      "theme",
+      !isDarkModeEnabled ? SupportedThemes.dark : SupportedThemes.light,
+    );
+    setIsDarkModeEnabled(!isDarkModeEnabled);
+    window.location.reload();
```

Comment:
I wasn't able to come up with a better solution for reloading the static css which has already been imported.

Reply:
Some good/bad news here:

Good:
- We have a dynamic index.html (aka custom/translatable tab title, description, meta attributes, etc) with helmet.
- CSS kind-of hot reload: 

![darkModeToggleFast](https://user-images.githubusercontent.com/11535726/109204319-7b7c8780-77a5-11eb-82a0-5e238457455b.gif)

Bad:

Rationale:
- React loads the css via js using webpack
  - React (with CRA) hides the webpack configuration. Alternatives: eject or use CRACO with a custom webpack config.
- For conditionally importing css we need to reload the component and import the file we want. That is the previous approach.
- For hot-reloading the css, the files should be in the <link ... href=... > tag
  - Since it's packed, we don't have a "blablabla.css" path
-  Ideas:
  - Using webpack for copying the files to the public folder
    - It requires replacing `react-scripts` by `craco` and handling our own webpack config. 
  - Using a simple `cp` for copying the files in each build from the @clr/ui folder.
    - This is the chosen option, but instead of just use  `cp`, I added a dev dep for making it work in other OS